src/DFMM.sol

```diff
@@ -101,6 +104,10 @@ contract DFMM is IDFMM {
 
         for (uint256 i = 0; i < params.tokens.length; i++) {
             if (params.tokens[i] != address(0)) {
```

Comment:
we were going to remove that check right?

Reply:
Of course, I just wanted to make a clean commit with all these checks being removed all at once.